### DISCUSSION TITLE
How to use Python to control the iCub

### DISCUSSION THREAD

**Discussion by pattacini** [robotology/community#373] [Show and tell]
@YANGGANG1971 asked in https://github.com/vvv-school/vvv18/issues/64 the following:

## Brief description

>Sorry, I am still unclear to know how to build a bidirectional communication interface between Python and Yarp.
I want to collect the sensory information from iCub using Python. And then, Python will process these data. Finally, the calculated outcome after the Python process will transfer to iCub and control the movement.  
Could you give me more details to learn about it?

**Comment by ddetommaso** [comment]
We're currently developing Python wrappers of YARP controllers and modules in this project

https://github.com/s4hri/pyicub
